frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/TraverseFromCurrentAction.java

```diff
@@ -12,14 +10,4 @@ public class TraverseFromCurrentAction extends BaseTraverseAction {
   public @Nullable String getNameOfBranchUnderAction(AnActionEvent anActionEvent) {
     return getCurrentBranchNameIfManaged(anActionEvent);
   }
-
-  @Override
-  @UIEffect
-  protected void onUpdate(AnActionEvent anActionEvent) {
```

Comment:
Assuming it's redundant indeed, https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1397#discussion_r1056356695

Reply:
oh, probably you are right -
- [ ] pls just make sure that you run ide locally, and the action looks correctly from the toolbar and from the find action dialog

Reply:
From toolbar: looks perfect

From find action dialog: looks that no branch-specific action is enabled 😯 

<img width="404" alt="image" src="https://user-images.githubusercontent.com/3383210/209437821-3aaf3e38-acec-41b4-bd25-d5cd3c31520d.png">

<img width="687" alt="image" src="https://user-images.githubusercontent.com/3383210/209437829-b272ca23-5c23-4791-8d06-69cc00eb8a2f.png">

<img width="680" alt="image" src="https://user-images.githubusercontent.com/3383210/209437833-785ee277-99a2-48a6-a707-c1ce61823cb4.png">


All greyed out... compare to the actions that are available:

<img width="402" alt="image" src="https://user-images.githubusercontent.com/3383210/209437837-3e95182e-830d-44fb-b2af-836fab0ce82b.png">

Is it even expected? can you see the same in your IDE? Lemme confirm this is also the case with current `master`...

Reply:
Woooow there's a lot of mess 😅 

<img width="1263" alt="image" src="https://user-images.githubusercontent.com/3383210/209438044-2c786b71-baf6-46b3-9717-092882ba43fe.png">

Reply:
The reason for Pull action being disabled is clearly wrong... and similarly with many other places. Not a thing of high priority to be fixed right now... lemme open a separate issue

Reply:
#1401

Reply:
I see some descriptions are wrong but in general the selected-based actions shall by disabled from the find action dialog

Reply:
Yeah... but this specific problem is about _Pull **current** branch_ action, not **selected** branch 🤔